server/api/internal/infrastructure/mongo/trigger.go

```
@@ -126,6 +126,13 @@
 	return r.client.RemoveOne(ctx, bson.M{"id": id.String()})
 }
 
+func (r *Trigger) writeFilter(filter any) any {
```

golangci-lint (1.64.8)

func `(*Trigger).writeFilter` is unused

(unused)